.circleci/config.yml

```diff
@@ -254,6 +254,25 @@ jobs:
           path: /tmp/test-reports
       - store_artifacts:
           path: /tmp/test-artifacts/conformance-coverage.html
+  test-lotus-soup:
```

Comment:
Looks like we didn't add this to any of the workflows at the bottom

Reply:
Good catch. Somehow I missed copying this from the test project.